src/classes/Callable_API.cls

```diff
@@ -32,7 +32,7 @@
 * @date 2019
 * @group API
 * @group-content ../../ApexDocContent/API.htm
-* @description Global API class that exposes a System.Callable interface for external
+* @description Global API class that exposes a System. interface for external
```

Comment:
What happened to the "callable" in the comments here and below?

Reply:
That's interesting. I am unsure why that is gone. I will resolve it.